lib/ui.js

```diff
@@ -153,7 +153,7 @@ module.exports = async (options, pkg) => {
 		{
 			type: 'confirm',
 			name: 'publishScoped',
-			when: isScoped(pkg.name) && !options.exists,
+			when: isScoped(pkg.name) && !options.exists && options.publish,
```

Comment:
We should also skip it when the package.json has `{"private": true}`. Get it from `pkg.private`.